experimental/devices/cap1188/cap1188.go

```diff
@@ -0,0 +1,554 @@
+// Copyright 2017 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package cap1188 controls a Microchip cap1188 device over I²C.
+
+// The device is a 8 Channel Capacitive Touch Sensor with 8 LED Drivers
+//
+// Datasheet
+//
+// The official data sheet can be found here:
+//
+// http://ww1.microchip.com/downloads/en/DeviceDoc/CAP1188%20.pdf
```

Comment:
They added a redirector from no-whitespace to whitespace, so for sack of "omg they can't type", let's put:
http://ww1.microchip.com/downloads/en/DeviceDoc/CAP1188.pdf